#!/usr/bin/env python 

PACKAGE = 'rulo_base'
from dynamic_reconfigure.parameter_generator_catkin import  *
gen = ParameterGenerator()
gen.add('main_brush' , int_t,   40, "Main Brush PWM", 40 , 0 , 80)
gen.add('side_brush' , int_t,   40, "Side Brush PWM", 40 , 0 , 80)
gen.add('vacuum' , int_t,   40, "Vacuum PWM", 40 , 0 , 80)
exit(gen.generate(PACKAGE,'rulo', 'BrushPWM'))